src/Node.hs

```diff
@@ -77,14 +77,16 @@ data ListenerAction packing m where
   -- | A listener that handles an incoming bi-directional conversation.
   ListenerActionConversation
     :: ( Packable packing snd, Unpackable packing rcv, Message rcv )
-    => (LL.NodeId -> ConversationActions snd rcv m -> m ())
```

Comment:
I'm skeptical of this because it gives great power to conversation listeners. Should they be able to send outside of the conversation itself? Start new conversations? Is there a particular use case that motivated this change?